### ISSUE TITLE
The results I got are very different from those in the paper

### ISSUE THREAD

**Issue by f-thirty** [stardist/stardist#138]
## ❓ Questions and Help

[文件.zip](https://github.com/stardist/stardist/files/6433838/default.zip)
The file is the predicted result. The operation steps are based on the training test on github step by step, but there are problems with the results. Can you see where the problem occurs?

**Comment by maweigert** [comment]
`If you open a new topic, please provide a clear and concise description to understand and ideally reproduce the issue you're having (e.g. including a code snippet, Python script, or Jupyter notebook).`

**Comment by f-thirty** [comment]
> `If you open a new topic, please provide a clear and concise description to understand and ideally reproduce the issue you're having (e.g. including a code snippet, Python script, or Jupyter notebook).`

[code.zip](https://github.com/stardist/stardist/files/6434116/code.zip)
We cut out the training code and test code according to the training steps provided, and the rest of the parameters are kept as defaults. We directly use this training file to train and test, and the result seems to be wrong.

**Comment by maweigert** [comment]
1) Looks like your input `example_image.tif` is a label image, not a raw input image
2) Could you run our [training notebook](https://github.com/stardist/stardist/blob/master/examples/3D/2_training.ipynb) (with setting `quick_demo = False`) and share the output?

**Comment by f-thirty** [comment]
> 1. Looks like your input `example_image.tif` is a label image, not a raw input image
> 2. Could you run our [training notebook](https://github.com/stardist/stardist/blob/master/examples/3D/2_training.ipynb) (with setting `quick_demo = False`) and share the output?

[data.zip](https://github.com/stardist/stardist/files/6434247/data.zip)
First of all, thank you for your reply. This is the image we used for training. The previous file put the predicted image. 
Our training code uses the code when quick_demo=false; but a problem I just found is that model.optimize_thresholds(X_val, Y_val) is not added at the end of the code. Could this be the cause of the poor results?

**Comment by maweigert** [comment]
Hard to debug from afar. If you run the training notebook and share the output, that would be the easiest.

**Comment by f-thirty** [comment]
> Hard to debug from afar. If you run the training notebook and share the output, that would be the easiest.
Sorry, I don't quite understand how to operate this shared output?

**Comment by maweigert** [comment]
> Sorry, I don't quite understand how to operate this shared output?

1. Run the [training notebook](https://github.com/stardist/stardist/blob/master/examples/3D/2_training.ipynb) with your data 
2. Export the notebook (including the output of the cells) via `Download As -> HTML` and share it here

**Comment by f-thirty** [comment]
[output.txt](https://github.com/stardist/stardist/files/6435124/output.txt)
Thank you for your help, this is the output after training, please take a look

**Comment by maweigert** [comment]
Please run the notebook as indicated above and export it as html.

**Comment by maweigert** [comment]
Closing due to inactivity
